wallet-api/src/Ledger/Value/TH.hs

```diff
@@ -60,33 +60,33 @@ makeLift ''Value
 The 'Value' type represents a collection of amounts of different currencies.
 
 We can think of 'Value' as a vector space whose dimensions are
-currencies. At the moment there is only a single currency (Ada), so 'Value'
-contains one-dimensional vectors. When currency-creating transactions are
-implemented, this will change and the definition of 'Value' will change to a
+currencies. At the moment there is only a single currency (Ada), so 'Value' 
+contains one-dimensional vectors. When currency-creating transactions are 
+implemented, this will change and the definition of 'Value' will change to a 
 'Map Currency Int', effectively a vector with infinitely many dimensions whose
 non-zero values are recorded in the map.
 
-To create a value of 'Value', we need to specifiy a currency. This can be done
-using 'Ledger.Ada.adaValueOf'. To get the ada dimension of 'Value' we use
+To create a value of 'Value', we need to specifiy a currency. This can be done 
+using 'Ledger.Ada.adaValueOf'. To get the ada dimension of 'Value' we use 
 'Ledger.Ada.fromValue'. Plutus contract authors will be able to define modules
 similar to 'Ledger.Ada' for their own currencies.
 
 -}
-
+   
 type CurrencyMap v = [(CurrencySymbol, v)]
```

Comment:
This should be `Map.Map CurrencySymbol Int`, and `lookup`, `these` etc. now live in `Ledger.Map` and `Ledger.These`

Reply:
See https://github.com/input-output-hk/plutus/pull/777/files#diff-1c1c3412fa62ce2041b9979203a12f71R66